.travis.yml

```diff
@@ -5,3 +5,4 @@ node_js:
   - "iojs"
 before_script: npm run lint
 script: ./node_modules/.bin/istanbul cover ./node_modules/.bin/nodeunit test && cat ./coverage/lcov.info | ./node_modules/.bin/coveralls && rm -rf ./coverage
+
```

Comment:
the build via travis was building so more or less seeing what retriggering would have done. Looks like coveralls is not talking with it.